doc/source/ray-dashboard.rst

```diff
@@ -0,0 +1,157 @@
+Check out the documentation to understand a rich set of features we offer.
+
+Getting Started
+---------------
+  
+Access Dashboard
+~~~~~~~~~~~~~~~~
+If you start Ray on your laptop, dashboard is running by default, and you can access it through its URL. 
+By default the URL is `localhost:8265`. Note that the port will be changed if the default port is not available.
+
+The URL is printed when Ray starts up.
+
+.. code-block:: text
+
+  INFO resource_spec.py:212 -- Starting Ray with 4.2 GiB memory available for workers and up to 2.12 GiB for objects. You can adjust these settings with ray.init(memory=<bytes>, object_store_memory=<bytes>).
+  INFO services.py:1093 -- View the Ray dashboard at localhost:8265
+
+Dashboard is also available when using autoscaler. Check out `Monitoring Autoscaler Status <autoscaling.html#monitoring-cluster-status>`_ for more details.
+
+Example Usage
+-------------
+
+To understand each components, look at `Dashboard Components <ray-dashboard.html#components>`_
+
+Debugging a Blocked Actor
+~~~~~~~~~~~~~~~~~~~~~~~~~
+If creating an actor requires resources (e.g., CPUs, GPUs, or other custom resources) that are not currently available, the actor cannot be created until those resources are added to the cluster or become available.
+This can cause an application to hang. To alert you to this issue, infeasible tasks are shown in red in the dashboard, and pending tasks are shown in yellow.
+You can find this information from the `Logical View` tab. 
+
+- **Infeasible Actor Creation** happens when an actor requires more resources than a Ray cluster can provide.
+- **Pending Actor Creation** happens when there is no available resource for this actor because it is already taken by other tasks / actors.
+
+.. code-block:: python
+  
+  ray.init(num_gpus=2)
+
+  @ray.remote(num_gpus=1)
+  class Actor1:
+      def __init__(self):
+          pass
+  
+  @ray.remote
+  class Actor2:
+      def __init__(self):
+          self.actor1_list = [Actor1.remote() for _ in range(4)]
+
+  @ray.remote(num_gpus=4)
+  class Actor3:
+      def __init__(self):
+          pass
+
+  actor2 = Actor2.remote()
+  actor3 = Actor3.remote()
+
+
+.. image:: https://raw.githubusercontent.com/ray-project/Images/master/docs/dashboard/dashboard-pending-infeasible-actors.png
+    :align: center
+
+This cluster has 2 GPUs. Actor1 requires 1 GPU to be created. Actor2 creates 4 Actor1. As a result of calling `Actor2.remote()`, 2 Actor1 creation tasks became pending. 
+
+You can also see it is infeasible to create Actor3 because it requires 4 GPUs which is bigger than the total gpus available in this cluster (2 GPUs). 
+
+Inspect Memory Usage
+~~~~~~~~~~~~~~~~~~~~
+The logical view displays the following information about memory usage
+
+- **Number of object IDs in scope**: number of local and remote objectIDs which actors can access.
```

Comment:
Is it the case that if any of these three numbers are increasing without bound, then there is a problem?

I still don't really understand these three definitions, so I'd suggest remove this section and just saying something like:
>**NumObjectIdsInScope**, **NumLocalObjects**, and **UsedLocalObjectMemory** refer to internal Ray concepts related to memory usage. If any of these numbers is increasing without bound, then there may be a memory leak in the application.

Reply:
If I am a developer who uses a dashboard, I'd prefer to know exactly what each metrics means. If you think this definition is confusing, and there is not way to improve, I think we should remove it from dashboard. Otherwise we should explain in the best details so users can use metrics for debugging.